MovieApp/MovieApp/NewsFeedTableViewController.m

```diff
@@ -0,0 +1,105 @@
+//
+//  NewsFeedTableViewController.m
+//  MovieApp
+//
+//  Created by user on 22/09/16.
+//  Copyright © 2016 internshipABH. All rights reserved.
+//
+
+#import "NewsFeedTableViewController.h"
+#import "FeedTableViewCell.h"
+#import "NewFeedsItem.h"
+#import "FeedDownloader.h"
+
+@interface NewsFeedTableViewController (){
+    NSMutableArray *news;
+    UISearchBar *searchBar;
+    UIBarButtonItem *leftButton;
+    UIBarButtonItem *rightButton;
+}
+
+@end
+
+@implementation NewsFeedTableViewController
+
++(NSURL *)getFeedsSource
+{
+    return [NSURL URLWithString:@"http://www.boxofficemojo.com/data/rss.php?file=topstories.xml"];
+}
```

Comment:
Please, move this method to separate configuration file. Same goes for all other configuration stuff you'll have.